src/CartingService/Application/Interfaces/IApplicationDbContext.cs

```diff
@@ -4,5 +4,5 @@ namespace CartingService.Application.Interfaces;
 
 public interface IApplicationDbContext
 {
-    public LiteDatabase Database { get; }
+    public ILiteDatabase Database { get; }
```

Comment:
I can see you changed it to the interface. But the question is still open. In your opinion is it ok to use reference to LiteDB in the Application layer?

Reply:
It's not a good decision. I think It would be better to move this interface into Infrastructure layer. Because literally it's infrastructure and only the repository depends on it.

Reply:
Cool, then proceed with this 😉

Reply:
So, I removed the interface from the Application layer and put it into Infrastructure\Persistance.